platform/rp2040/make/Makefile

```diff
@@ -5,6 +5,7 @@ BIN = free_dap_rp2040
 ##############################################################################
 .PHONY: all directory clean size release
 
+BIN2UF2 = ../tools/bin2uf2
```

Comment:
bin2uf is not really a part of the project, I just don't have a better place for it. I just place it long some of the projects for now. So, referencing it like this is not going to work. On my system I built it once and put it into a place accessible from the system PATH.

It is better to just do BIN2UF=bin2uf and then specify it externally for the build "make BIN2UF=./tools/bin2uf" or whatever.

But eventually I hope to move it from here to some better place. May be at least root of the project for now.

Reply:
I did move it here https://github.com/ataradov/tools